server/src/test/java/org/elasticsearch/bwcompat/RecoveryWithUnsupportedIndicesIT.java

```diff
@@ -81,16 +81,16 @@ protected Settings prepareBackwardsDataDir(Path backwardsIndex) throws IOExcepti
         return builder.build();
     }
 
-    public void testUpgradeStartClusterOn_0_20_6() throws Exception {
-        String indexName = "unsupported-0.20.6";
+    public void testUpgradeStartClusterOn_2_4_5() throws Exception {
+        String indexName = "unsupported-2.4.5";
```

Comment:
Are any of the other `unsupported-x.x.x.zip` files used?

Reply:
not as far as I can see (running the tests will confirm). I initially wanted to remove those in a follow-up, but I think we can do it as well in this PR.